parts/linux/cloud-init/artifacts/components.json

```diff
@@ -71,18 +71,10 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cns:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.4.52",
-        "v1.5.26",
```

Comment:
1.5.26 is also still used in AKS (for k8s 1.28, until we finish rolling out 1.5.32)

Reply:
Sounds good, I'll leave this PR open for when we are ready to remove 1.5.26

Reply:
1.5.32 has started rolling out to AKS prod so we can follow up here in ~2 weeks